snuba/datasets/metrics_aggregate_processor.py

```diff
@@ -0,0 +1,135 @@
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Any, Mapping, Optional
+
+from snuba.clickhouse.value_types import (
+    ClickhouseInt,
+    ClickhouseNumArray,
+    ClickhouseUnguardedExpression,
+)
+from snuba.consumers.types import KafkaMessageMetadata
+from snuba.processor import (
+    AggregateInsertBatch,
+    MessageProcessor,
+    ProcessedMessage,
+    _ensure_valid_date,
+)
+
+
+class MetricsAggregateProcessor(MessageProcessor, ABC):
+    GRANULARITIES_SECONDS = [10, 3600, 3600 * 24]
```

Comment:
I was probably not clear with my comments the first time around. These are Granularity buckets in seconds. I was suggesting a change along the following lines where the time periods were defined as constants so that the code is easy to read. But it is not a blocker for me.
`
GRANULARITY = [10SEC, 1HOUR, 1DAY]
`

Reply:
I don't feel strongly, but I would like for one more person to agree with you before making this change as I don't think it increases clarity for me